db/changelog/001-create-timeEntry-and-timePeriodType-table.json

```diff
@@ -0,0 +1,18 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "002-create-timeEntry-and-timePeriodType-table",
+        "author": "Callisto Dev",
```

Comment:
Should this not be the actual developer's name

Reply:
Oh yeah good spot! Although it seems fairly redundant considering we’ll have that recorded in source control 🤔 docs suggest it’s required though but if we _can_ get rid of it that’d be my vote. Not sure what it adds.

Reply:
It depends. How we want it to be. I have used to follow the team name.

Reply:
I don't have a strong opinion on this, as long as it is consistent across the board

Reply:
I have included dev name i.e. my name